wiki-site/docs/Data Acquisition/Custom DAQ PCB Boards/DAQ Revision 1/Revision 1.md

```diff
@@ -0,0 +1,260 @@
+
+### BOOT0
+The microcontroller's `BOOT0` pin dictates the firmware that runs on the board.
+
+- When tied to `LOW`, the board will run on the firmware that is on its flash memory.
+- When tied to `HIGH` (that is, `VCC`), then it will run the system memory program.
+
+However, since the board has SWD connectors the board can be programmed regardless of the state of `BOOT0`
+
+!!! warning
+    From personal experience this has not been the case, but keeping the `BOOT0` pin tied to `GND` seems to always work.
+    
+If programming via USB-to-UART, `BOOT0` becomes more relevant and you will need to tie it to `GND` to support the USB-to-UART programming feature. To ensure the functionality of both USB-to-UART and SWD programming, the `BOOT0` was put into an SPDT switch that changes between `GND` and `VCC`.
+
+### NRST
+`NRST` is a reset pin tied to a button that can reset the microcontroller when NRST is set to `LOW`.
+
+!!! danger
+
+    This pin is not advised to be left floating as that could lead to accidental, spurious resets of the microcontroller. 
+
+A typical 4-pin button was placed that connects `GND` and `NRST`. A 100 nF capacitor is also added between `NRST` and `GND` to debounce the switch and ensure that voltage spikes do not accidentally toggle the NRST.
+
+### VCAP 1 and VCAP 2
+These are capacitors that are connected to the **internal** voltage regulator of the MCU. This voltage regulator is primarily for powering the core and internal components of the MCU. The `VCAP` pins act as decoupling capacitors for this voltage regulator. Based on stm32 PCB guidelines[^1] these values should be around 2.2u capacitors that have a low ESR.
+
+!!! note
+
+    The actual PCB uses 22 uF. This was a typo when making the schematic, but the board still functions as intended.
+
+### PDR
+This pin is used to either enable or disable the Power Down on Reset (PDR).
+
+PDR serves as a reset function when the supply voltage drops below a certain threshold level. Typically, this is 3.3 volts. The whole reason for this is to ensure that the microcontroller will only function as intended. Low voltage supplies can make the MCU unpredictable. Otherwise, if left on, the system may behave unpredictably.
+
+- When set to `HIGH`, the PDR features are enabled.
+- When set to `LOW`, all the features are disabled.
+
+### Vref
+This is the voltage reference used for the built-in ADCs on the board and must be tied into a positive voltage that is relatively clean from noise.
+
+### HSE
+High-Speed External (HSE) is not required for the board, but is something that most boards will have. The point of this pin is to add an external clock to the microcontroller to ensure a more precise and accurate clock.
+Note that this external oscillator needs to be within a certain range of frequencies and this differs for each board[^2]. The stm32 F429 that is used for this DAQ can take an oscillator frequency ranging from 4 to 26 MHz.
```

Comment:
> ranging from 4 to 6 MHz.

How did you derive these values?

Reply:
these value were found in both the data sheet and in the .ioc file. in clock configuration the input values for the HSE are in these range